src/main/java/com/synopsys/integration/alert/channel/jira/common/util/JiraIssueHandler.java

```diff
@@ -69,8 +69,8 @@ public JiraIssueHandler(Gson gson, JiraTransitionHandler jiraTransitionHandler,
     public abstract String getIssueCreatorFieldKey();
 
     @Override
-    protected Optional<IssueResponseModel> createIssue(IssueConfig issueConfig, IssueTrackerRequest request)
-        throws IntegrationException {
+    // TODO this does not need to be Optional
```

Comment:
Would the change be big to resolve this todo right now?

Reply:
It's scope-creep for a code-frozen version. Otherwise I definitely would have made this change.